pages/k8s/cis-compliance.md

```diff
@@ -14,13 +14,78 @@ toc: False
 ---
 
 The **Center for Internet Security (CIS)** maintains a
-[Kubernetes benchmark][cis-benchmark] that is helpful to ensure clusters are
+[Kubernetes benchmark][cis-benchmark] that helps ensure clusters are
 deployed in accordance with security best practices. **Charmed Kubernetes**
 includes support for the [kube-bench][] utility, which reports how well a
-cluster complies with this benchmark. This page details how to run these
-tests.
+cluster complies with this benchmark. This page highlights compliance
+requirements as well as details on running the benchmark and analyzing test
+results.
 
-## Run kube-bench
+## Compliance Requirements
+
+**Charmed Kubernetes** 1.19 is compliant by default. Key configuration changes
+from previous releases include the following:
+
+### `kubernetes-master`
+
+#### Set `authorization.mode = Node,RBAC`
+
+`kube-apiserver` must not include `AlwaysAllow` as an authorization mode.
+This is set by the `authorization-mode` config option on the `kubernetes-master`
+charm (`Node,RBAC` by default).
+
+#### Set `encryption-provider-config`
+
+`kube-apiserver` must set `encryption-provider-config` to the path of a valid
+`EncryptionConfig` manifest
+(`/var/snap/kube-apiserver/common/encryption/encryption_config.yaml` by
+default).
+
+#### Disable `insecure-bind-address` and `insecure-port`
+
+`kube-apiserver` must not respond to requests over an insecure address.
+
+#### Enable `NodeRestriction` and `PodSecurityPolicy` plugins
+
+`kube-apiserver` must enable the `NodeRestriction` and `PodSecurityPolicy`
+admission control plugins.
+
+#### Disable `profiling`
+
+`kube-apiserver`, `kube-controller-manager`, and `kube-scheduler` must set
+`profiling=False`.
+
+#### Set `terminated-pod-gc-threshold`
+
+`kube-controller-manager` must set a value for `terminated-pod-gc-threshold`
+(12500 by default).
+
+#### Disable `token-auth-file`
+
+`kube-apiserver` must not use file-based authentication. **Charmed Kubernetes**
+now deploys a webhook authentication service that compares API requests to
+Kubernetes secrets. If needed, any existing entries in `known_tokens.csv` are
+migrated to secrets on charm upgrade.
+
+### `kubernetes-worker`
+
```

Comment:
worth linking to the config docs? https://ubuntu.com/kubernetes/docs/charm-kubernetes-worker#configuration

Reply:
Similar to the k8s-master comment, there's nothing the user could change here with k8s-worker charm config.  We're baking these values into the charm.  Maybe worth linking the k8s-worker charm page, but i don't see a benefit in linking to the #config anchor.  Thoughts?

Reply:
ok, I didn't realise they weren't configurable